plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/DefaultMapViewPart.java

```diff
@@ -206,4 +211,21 @@ public void menuAboutToShow( IMenuManager mgr ) {
         }
     }
 
+    @Override
+    public UDIGDropHandler getDropHandler() {
+        return ((UDIGControlDropListener) dropTarget.listener).getHandler();
+    }
+
+    //
+    // helper method for ToolManager
```

Comment:
You may use the /** comment section instead of the // ?

Reply:
IMHO we can remove the comments here